frontend/src/pages/Exercise/index.js

```diff
@@ -95,6 +107,9 @@ export class Exercise extends Component {
 
   handleSubmit = () => {
     const data = Object.keys(this.state).map(answer => this.state[answer]);
+    const { exercise } = this.props;
+    console.log(exercise);
+    this.props.startCountdown(+exercise.lengthMinutes);
```

Comment:
Why are you casting `lengthMinutes` into a number? Is it not already?

Reply:
it was a given as a string in the mock data API but is in fact a number in the BE API so this can be cleaned up